src/Checkbox/Checkbox.jsx

```diff
@@ -77,11 +81,13 @@ Checkbox.propTypes = {
   onChange: PropTypes.func,
   checked: PropTypes.bool,
   size: PropTypes.string,
-  variant: PropTypes.string
+  variant: PropTypes.string,
+  disabled: PropTypes.bool
 }
 
 Checkbox.defaultProps = {
   color: 'brand',
   size: 'medium',
-  variant: 'labelSmall'
+  variant: 'labelSmall',
```

Comment:
the new `base14Light` variant should be equivalent

Reply:
29d79de4d68c55cac0aa453273d77d401635756b